### DISCUSSION TITLE
Available NETCONF plugins for Nornir

### DISCUSSION THREAD

**Discussion by fwclements** [nornir-automation/nornir#626] [Q&A] [answered]
Does anyone have a lead on a netconf plugin?  It's unfortunate that it's now missing from Nornir and it's other frameworks.

**Comment by lemontree61089** [comment, marked as answer]
It already exists and works well: https://pypi.org/project/nornir-scrapli/

You can find the list of plugins here: https://nornir.tech/nornir/plugins/

**Comment by ktbyers** [comment]
There is also this...though not sure the status of it:

https://github.com/nornir-automation/nornir_netconf/tree/first
